Trainee/Reviews/Reviews.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { useState } from "react";
+import { FaStar } from "react-icons/fa";
+import axios from "../../../api/axios";
+import ViewReviews from "./ViewReviews/ViewReviews";
+import { Scrollbars } from "react-custom-scrollbars";
+import "./Reviews.css";
+
+const reviewsUrl = "/review";
+
+export default function Review({ training, traineeName }) {
+  const [currentValue, setCurrentValue] = useState(0);
+  const [hoverValue, setHoverValue] = useState(undefined);
+  const [trainingType, setTrainingType] = useState("");
+  const [review, setReview] = useState("");
+  const date = new Date().toLocaleString("en-us", {
```

Comment:
should you read the Locale from browser?

Reply:
??